File: clients/Scala/src/main/scala/MyStrategy.scala
class MyStrategy {
  def getAction(playerView: model.PlayerView, debugInterface: Option[DebugInterface]): model.Action = model.Action(Map())
  def debugUpdate(playerView: model.PlayerView, debugInterface: DebugInterface) {
    debugInterface.send(model.DebugCommand.Clear())
    debugInterface.getState()
  }
}

File: clients/FSharp/MyStrategy.fs
namespace Aicup2020

open Aicup2020.Model

type MyStrategy() =
    member this.getAction(playerView: PlayerView, debugInterface: Option<DebugInterface>): Action =
        { EntityActions = Map.empty }

    member this.debugUpdate(playerView: PlayerView, debugInterface: DebugInterface) =
        debugInterface.send (DebugCommand.Clear(new DebugCommandClear()))
        debugInterface.getState () |> ignore
